common/type_system/Type.cpp

```diff
@@ -936,12 +936,16 @@ std::string BasicType::diff_impl(const Type& other_) const {
 // Bitfield
 /////////////////
 
-BitField::BitField(TypeSpec type, std::string name, int offset, int size)
-    : m_type(std::move(type)), m_name(std::move(name)), m_offset(offset), m_size(size) {}
+BitField::BitField(TypeSpec type, std::string name, int offset, int size, bool skip_in_decomp)
+    : m_type(std::move(type)),
+      m_name(std::move(name)),
+      m_offset(offset),
+      m_size(size),
+      m_skip_in_static_decomp(skip_in_decomp) {}
 
 bool BitField::operator==(const BitField& other) const {
   return m_type == other.m_type && m_name == other.m_name && m_offset == other.m_offset &&
-         other.m_size == m_size;
+         m_size == other.m_size;
```

Comment:
No, it's specifically *not* checked for in normal structures.

Reply:
ok, makese sense. I think if you define a type twice, and only have `do-not-decompile` in the second definition it will be silently ignored, but that's not the end of the world.

Reply:
Yep, seems intentional https://github.com/open-goal/jak-project/blame/77c0c97fa0a95a342b6ab231c9a153415a6dccd0/common/type_system/Type.cpp#L117